libs/blocks/tabs/tabs.css

```
@@ -323,6 +344,61 @@
   border: none;
 }
 
+.tabs.radio {
+  background: none;
+}
+
+.tabs.radio .tab-content {
+  border: none;
+}
+
+.tabs.radio .tab-paddles {
+  display: none;
+}
+
+.tabs.radio div[role="tablist"] .tab-list-container {
+  border: 1px solid #d9d9d9;
+  border-radius: 24px;
+  padding: var(--spacing-xs) var(--spacing-s) var(--spacing-s);
+  font-size: var(--type-body-s-size);
+  color: var(--text-color);
+  flex-direction: column;
+  align-items: start;
```

We can remove the `align-items: start;` from here. It will default to `align-items: stretch;` which will result in the same behaviour in this scenario.
```suggestion
  flex-direction: column;
```